data/projects/implement_metrics_for_coala.md

```diff
@@ -1,34 +1,43 @@
-There is more to software quality than just passing builds -
-coala should support generating metrics for your code.
+There is more to software quality than just passing builds. coala should
```

Comment:
Also you say coala should support metrics before talking about metrics and why coala should support them